### PR TITLE
Dependabot/npm and yarn/liquidjs 10.0.0

### PR DESCRIPTION
**Pull request by Winlove0710**
### Why:

Closes ISSUE

### Check off the following:

- [x] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -47,7 +47,7 @@
         "javascript-stringify": "^2.1.0",
         "js-cookie": "^3.0.1",
         "js-yaml": "^4.1.0",
```
Comment (Winlove0710) [inline package-lock.json:49]: >        "liquidjs": "10.0.0",
         "lodash": "^4.17.21",
         "lodash-es": "^4.17.21",
         "lunr": "^2.3.9",
        "lunr-languages": "^1.9.0",
        "mdast-util-from-markdown": "^1.2.0",
        "mdast-util-to-string": "^3.1.0",
        "morgan": "^1.10.0",
        "msgpack5rpc": "^1.1.0",
        "next": "12.2.4",
        "parse5": "7.1.1",
        "port-used": "^2.0.8",
        "react": "^17.0.2",
        "react-dom": "^17.0.2",
        "react-markdown": "^8.0.3",
        "react-syntax-highlighter": "^15.5.0",
        "rehype-autolink-headings": "^6.1.1",
        "rehype-highlight": "^6.0.0",
        "rehype-raw": "^6.1.1",
        "rehype-slug": "^5.0.1",
        "rehype-stringify": "^9.0.3",
        "remark-code-extra": "^1.0.1",
        "remark-gemoji-to-emoji": "^1.1.0",
        "remark-gfm": "^3.0.1",
 @@ -14022,34 +14022,35 @@
      "resolved": "https://registry.npmjs.org/path-key/-/path-key-4.0.0.tgz",
      "integrity": "sha512-haREypq7xkM7ErfgIyA0z+Bj4AGKlMSdlQE2jvJo6huWD1EdkKYV+G/T4nq0YEF2vgTT8kqMFKo1uHn950r4SQ==",
      "dev": true,
      "engines": {
        "node": ">=12"
      },
      "funding": {
        "url": "https://github.com/sponsors/sindresorhus"
      }
    },
    "node_modules/lint-staged/node_modules/strip-final-newline": {
      "version": "3.0.0",
      "resolved": "https://registry.npmjs.org/strip-final-newline/-/strip-final-newline-3.0.0.tgz",
      "integrity": "sha512-dOESqjYr96iWYylGObzd39EuNTa5VJxyvVAEm5Jnh7KGo75V43Hk1odPQkNDyXNmUR6k+gEiDVXnjB8HJ3crXw==",
      "dev": true,
      "engines": {
        "node": ">=12"
      },
      "funding": {
        "url": "https://github.com/sponsors/sindresorhus"
       }
     },
     "node_modules/liquidjs": {

File: package-lock.json [hunk 2]
```diff
@@ -14042,14 +14042,15 @@
       }
     },
     "node_modules/liquidjs": {
-      "version": "9.22.1",
-      "license": "MIT",
```
Comment (Winlove0710) [inline package-lock.json:14046]: >       "version": "10.0.0",
       "resolved": "https://registry.npmjs.org/liquidjs/-/liquidjs-10.0.0.tgz",
       "integrity": "sha512-uY3fKqBSRmnat0wB5qWk0W5N5iT78OUhuIXNRiMfhYQ3p35al9mEWKQ68/Z+WD54V2upaxEM8nHaZ6o1B8WCMA==",
       "bin": {
         "liquid": "bin/liquid.js",
         "liquidjs": "bin/liquid.js"
       },
       "engines": {

File: package-lock.json [hunk 3]
```diff
@@ -14042,14 +14042,15 @@
...
+      "version": "10.0.0",
+      "resolved": "https://registry.npmjs.org/liquidjs/-/liquidjs-10.0.0.tgz",
+      "integrity": "sha512-uY3fKqBSRmnat0wB5qWk0W5N5iT78OUhuIXNRiMfhYQ3p35al9mEWKQ68/Z+WD54V2upaxEM8nHaZ6o1B8WCMA==",
       "bin": {
         "liquid": "bin/liquid.js",
         "liquidjs": "bin/liquid.js"
       },
       "engines": {
```
Comment (Winlove0710) [inline package-lock.json:14052]: >       "node": ">=14"
       },
       "funding": {
         "type": "opencollective",
 @@ -30731,7 +30732,9 @@
       }
     },
     "liquidjs": {

File: package-lock.json [hunk 4]
```diff
@@ -30731,7 +30732,9 @@
       }
     },
     "liquidjs": {
```
Comment (Winlove0710) [inline package-lock.json:30734]: >       "version": "10.0.0",
       "resolved": "https://registry.npmjs.org/liquidjs/-/liquidjs-10.0.0.tgz",
       "integrity": "sha512-uY3fKqBSRmnat0wB5qWk0W5N5iT78OUhuIXNRiMfhYQ3p35al9mEWKQ68/Z+WD54V2upaxEM8nHaZ6o1B8WCMA=="
     },
     "listr2": {
       "version": "4.0.5",

File: package.json [hunk 5]
```diff
@@ -49,7 +49,7 @@
     "javascript-stringify": "^2.1.0",
     "js-cookie": "^3.0.1",
     "js-yaml": "^4.1.0",
```
Comment (Winlove0710) [inline package.json:52]: >     "liquidjs": "10.0.0",
     "lodash": "^4.17.21",
     "lodash-es": "^4.17.21",
     "lunr": "^2.3.9",

File: package-lock.json [hunk 6]
```diff
@@ -30333,7 +30334,9 @@
...
```
Comment (Winlove0710) [inline package-lock.json:30336]: >       "version": "10.0.0",
       "resolved": "https://registry.npmjs.org/liquidjs/-/liquidjs-10.0.0.tgz",
       "integrity": "sha512-uY3fKqBSRmnat0wB5qWk0W5N5iT78OUhuIXNRiMfhYQ3p35al9mEWKQ68/Z+WD54V2upaxEM8nHaZ6o1B8WCMA=="
     },
     "listr2": {
       "version": "4.0.5",

File: package-lock.json [hunk 7]
```diff
@@ -48,7 +48,7 @@
         "javascript-stringify": "^2.1.0",
         "js-cookie": "^3.0.1",
         "js-yaml": "^4.1.0",
```
Comment (Winlove0710) [inline package-lock.json:50]: >        "lodash": "^4.17.21",
         "lodash-es": "^4.17.21",
         "lowdb": "5.0.5",
 @@ -14029,14 +14029,15 @@
       }
     },
     "node_modules/liquidjs": {

File: package-lock.json [hunk 8]
```diff
@@ -14029,14 +14029,15 @@
       }
     },
     "node_modules/liquidjs": {
```
Comment (Winlove0710) [inline package-lock.json:14031]: >       "version": "10.0.0",
       "resolved": "https://registry.npmjs.org/liquidjs/-/liquidjs-10.0.0.tgz",
       "integrity": "sha512-uY3fKqBSRmnat0wB5qWk0W5N5iT78OUhuIXNRiMfhYQ3p35al9mEWKQ68/Z+WD54V2upaxEM8nHaZ6o1B8WCMA==",
       "bin": {
         "liquid": "bin/liquid.js",
         "liquidjs": "bin/liquid.js"
       },
       "engines": {

File: package-lock.json [hunk 9]
```diff
@@ -14029,14 +14029,15 @@
...
-      "version": "9.22.1",
-      "license": "MIT",
+      "version": "10.0.0",
+      "resolved": "https://registry.npmjs.org/liquidjs/-/liquidjs-10.0.0.tgz",
+      "integrity": "sha512-uY3fKqBSRmnat0wB5qWk0W5N5iT78OUhuIXNRiMfhYQ3p35al9mEWKQ68/Z+WD54V2upaxEM8nHaZ6o1B8WCMA==",
       "bin": {
         "liquid": "bin/liquid.js",
         "liquidjs": "bin/liquid.js"
       },
       "engines": {
```
Comment (Winlove0710) [inline package-lock.json:14039]: >         "node": ">=14"
       },
       "funding": {
         "type": "opencollective",
 @@ -30333,7 +30334,9 @@
       }
     },
     "liquidjs": {

Comment (Tomasmajda) [reply package-lock.json:14039]: "funding": {
     "type": "opencollective",

Comment (Tomasmajda) [reply package-lock.json:14039]: [package-lock.json](https://github.com/github/docs/pull/22886/files/a99fcd405cf7d735ca9f84f92cd04ca2c69b5bdc#diff-053150b640a7ce75eff69d1a22cae7f0f94ad64ce9a855db544dda0929316519)

Comment (Tomasmajda) [reply package-lock.json:14039]: unlock

**Pull request closed without merging**